docs/src/cli/version.md

```diff
@@ -42,3 +42,11 @@ tree-sitter version
 ### `-p/--grammar-path <PATH>`
 
 The path to the directory containing the grammar.
+
+### `--bump`
```

Comment:
I'm not a fan of this feature at all, to be honest -- way too brittle, and it has _never_ worked correctly for me. It also relies on external tooling for some of the  bindings, which needs to be documented.

(Again, parser maintainers get these whether they want to or not -- in particular whether they use or know the language or not.)

Reply:
(Not a blocker for this PR, of course, which should document all existing flags. But now would be a great time to deprecate this _if_ we want to remove it ;))